### ISSUE TITLE
NTR: Chest imaging: boot-shaped cardiac silhouette (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9031]
New term request Chest imaging: boot-shaped cardiac silhouette (nadea3_191112211901): Heart; CXR; Chest X-ray; heart silhouette has an upturned apex and a concave main pulmonary artery segment  UMLS: 2073425

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000145
- label: Boot-shaped cardiac silhouette
- parents: [Abnormal heart morphology (HP:0001627)](https://hpo.jax.org/app/browse/term/HP:0001627)
- pmid: 
- def: An abnormal form of the heart seen upon chest radiography in which the heart displays an upturned apex and slight concave pulmonary artery segment. This configuration is said to resemble a boot.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9031
- synonyms: Boot shaped cardiac silhouette(NONE)
